src/Payments.sol

```diff
@@ -672,80 +725,15 @@ contract Payments is
         );
     }
 
-    function updateOperatorLockupTracking(
```

Comment:
Don't need these anymore as allowance tracking and updates now happen in the settleAccountLockupBeforeAndAfterForRail modifier.